staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -123,4 +123,6 @@ type KMSConfiguration struct {
 	CacheSize int32
 	// endpoint is the gRPC server listening address, for example "unix:///var/run/kms-provider.sock".
 	Endpoint string
+	// Timeout for gRPC calls to kms-plugin (ex. 5s).
+	Timeout metav1.Duration
```

Comment:
should be `*metav1.Duration` so we can tell the difference between "0" and unspecified